src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -499,17 +499,20 @@ public static LocalDateTime parseLocalDateTime(String dateTimeString, String pat
     }
 
     /**
-     * Parses a `LocalDateTime` object from separated date, hour and minute strings.
+     * Parses a {@code LocalDateTime} object for session form from separated date, hour and minute strings.
+     * Example: If date is Tue, 01 Apr, 2014, hour is 23, min is 59.
+     * Returns {@code null} on error.
```

Comment:
You can delete this `Returns` line, there's already a `@return` annotation below that covers this.